services/QuillLMS/app/models/concerns/public_progress_reports.rb

```diff
@@ -249,9 +250,9 @@ def format_concept_results(activity_session, concept_results)
   end
   # rubocop:enable Metrics/CyclomaticComplexity
 
-  def get_key_target_skill_concept_for_question(concept_results)
+  def get_key_target_skill_concept_for_question(concept_results, is_evidence)
```

Comment:
I try to avoid passing booleans as a parameter as it usually is a code smell hiding complexity.

One refactor would be passing the activity session instead:
```
def get_key_target_skill_concept_for_question(concept_results, activity_session)
...
   name:  activity_session.is_evidence? ?  'Writing with Evidence' : 'Conventions of Language',
      correct: get_score_for_question(concept_results) > 0
    }
```
and adding a method to activity_session.rb

```
def is_evidence?
  classification.key == ActivityClassification::EVIDENCE_KEY
end
```
This gets rid of some of the duplication as well as cleaning up the law of demeter issue.